README.md

```diff
@@ -66,8 +68,10 @@ Sample (counting event) with metadata
       "id":"79031920-6239-471e-a3d7-f241b7753fd0",
       "name":"streamname"
    },
-   "version":"4.0",
-   "eventSchema": "https://swarm-analytics.com/schema/event/"
+   "version":"5.0",
+   "eventSchema": "https://swarm-analytics.com/schema/event/",
```

Comment:
We could change the schema in one go. No hard feelings, but we don't even control the domain anymore / nothing is there.

Reply:
We discussed offline to remove it?